resources.md

```diff
@@ -0,0 +1,62 @@
+# Useful resources, research, and quick links
+
+## Purpose  
+Serve as a quick resource for the sig to grab links that we work with at high frequency, reference, other orgs research. This is not official guidance. 
+
+### CNCF Quick Links
+
+- [TOC repo](https://github.com/cncf/toc)
+  - [Principles](https://github.com/cncf/toc/blob/master/PRINCIPLES.md)
+  - [What type of goverance is my project expected to follow?](https://github.com/cncf/toc/blob/master/FAQ.md#what-type-of-governance-is-my-cncf-project-expected-to-follow)
+- [Due Diligence guidelines](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md#project)
+- [Due Diligence Template](https://github.com/cncf/toc/blob/master/process/dd-review-template.md#project)
+- [Graduation Requirements](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage)
+- [Sandbox->Incubation Requirements](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#incubating-stage)
+[Sandbox Annual Review has related questions](https://github.com/cncf/toc/blob/master/process/sandbox-annual-review.md#annual-review-contents)
+
+### Governance 
+- https://sustainers.github.io/governance-readiness/  
```

Comment:
is this missing a [TEXT] to refer to it

Reply:
wow, we need to add a lot of resources here.  that's a TODO for me.